utils.py

```diff
@@ -453,3 +454,17 @@ def get_all_language_codes_and_names():
 def unescape_encoded_uri_component(escaped_string):
     """Unescape a string that is encoded with encodeURIComponent."""
     return urllib.unquote(escaped_string).decode('utf-8')
+
+
+cache_slug = None  # pylint: disable=invalid-name
+def get_cache_slug():
```

Comment:
Is this in tests? If they aren't testing the utils method then maybe you
can swap out the utils method.

On Sunday, July 3, 2016, Vishal Gupta <notifications@github.com> wrote:

> In utils.py
> <https://github.com/oppia/oppia/pull/2208#discussion_r69388999>:
>
> > @@ -453,3 +454,17 @@ def get_all_language_codes_and_names():
> >  def unescape_encoded_uri_component(escaped_string):
> >      """Unescape a string that is encoded with encodeURIComponent."""
> >      return urllib.unquote(escaped_string).decode('utf-8')
> > +
> > +
> > +cache_slug = None  # pylint: disable=invalid-name
> > +def get_cache_slug():
>
> There's an issue: I am currently caching the cache_slug, let's say
> feconf.DEV = True and the cache_slug is generated. Then, I swap it with
> feconf.DEV = False and request the cache slug again, and it returns the old
> value which is wrong.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/oppia/oppia/pull/2208/files/126df3730393afd91f660eb4f210060541710fee#r69388999>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKFeyjT0pD3TzK4qclTyHCjINXPJn_CCks5qR45bgaJpZM4JDv96>
> .
>